wasi/type_size.go

```diff
@@ -0,0 +1,6 @@
+package wasi
+
+// Constants of the size of frequent types in bytes
+const (
+	SIZE_UINT32 = 4
```

Comment:
we don't need to define the obvious consts like this

Reply:
Removed.